### PR TITLE
Add a luarock generation script

### PR DESCRIPTION
**Pull request by nathanmonteleone**

### CODE REVIEW

File: ngx_lua_datadog_TEMPLATE.rockspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package = "ngx_lua_datadog"
```
Comment (nathanmonteleone) [inline ngx_lua_datadog_TEMPLATE.rockspec:1]: So here's the deal -- rockspecs are *required* to have the version stamped into the filename, as well as in the file itself.  IMO this is redundant, but it is what it is.  So I figured it was best to make a template file and a script to `sed` and copy it to have the version in all the right places...

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ tags = "country:china,datacenter:asia,account:mass_market"
 logger:count("another_hit", 1, 1, tags)
 ```
 
+## Luarock creation
 
-
-
-
+1. Run `sh scripts/build_rock.sh <version>` where version corresponds to the tag you want to build.  Ex. `sh scripts/build_rock.sh v0.1.0`.
+2. Upload the .src.rock file to the server (TBD)
```
Comment (nathanmonteleone) [inline README.md:78]: TBD because you're going to need an API key to do this...  I'm not sure if we should make a "sifi" account on luarocks.org or what.

Comment (nathanmonteleone) [reply README.md:78]: or just a ngx_lua_datadog "maintainer" key?

**Review by mcmoyer — APPROVED** [review]

**Review by joshuawscott — APPROVED** [review]
I like the templating thing to get around the goofy number in a file scheme

**Pull request merged**